pkgs/os-specific/linux/corefreq/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  kernel,
+  hostPlatform,
+  # See the official readme for a list of optional flags:
+  # https://github.com/cyring/CoreFreq/blob/master/README.md
+  extraFlags ? [ ],
+}:
+
+stdenv.mkDerivation rec {
+  pname = "corefreq";
+  version = "1.98.0";
```

Comment:
Could you bump this to 1.98.4 for compatibility with Linux 6.11?

Reply:
Done